src/components/Sender.tsx

```diff
@@ -149,12 +157,12 @@ const Sender = () => {
         {chains.map((chain) => renderAccounts(chain))}
       </Select>
       <div className={classes.accountCompanion}>
-        {derivedAccount ? (
-          <Account
+        {companionAccount && senderCompanionAccountBalance ? (
```

Comment:
`senderCompanionAccountBalance` can be `0`, but I guess since it's a `BN` it passes the check, right? That said I wonder if it wouldn't be better to explicitly compare the balance with `null`?

Reply:
> if it wouldn't be better to explicitly compare the balance with null

sure!